event_dictionary/v0.3_DRAFT/examples/collectionExerciseUpdate.example.json

```diff
@@ -13,7 +13,16 @@
     "collectionExerciseUpdate": {
       "collectionExerciseId": "3883af91-0052-4497-9805-3238544fcf8a",
       "surveyId": "3883af91-0052-4497-9805-3238544fcf8a",
-      "name": "velit"
+      "name": "velit",
+      "reference": "MVP012021",
+      "startDate": "2021-09-17T23:59:59.999Z",
+      "endDate": "2021-09-27T23:59:59.999Z",
+      "metadata": {
```

Comment:
this hand-written example will be overwritten the next time the machine generation gets run.

To write your own "custom" examples, it's often best to put stuff into `polish_example_json.py` so that you have full control over how the example looks.

Reply:
ah yes, have added that now 👍